### DISCUSSION TITLE
v0.6.1

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#125] [Announcements]
1. 修复了双光标的问题

    - 现在可以应用于所有游戏
    - 现在可以在联机游戏中使用
    - 使用中遇到问题请提交 issue

2. 添加关闭窗口时最小化到系统托盘的功能（感谢 @JeffersonQin）

3. 修复了在v1903之前的系统上无法使用 GDI 捕获的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.6.1'>v0.6.1</a>.</em>

**Comment by RyzenAdvanced** [comment]
Hopefully,  development team can try add these new DirectML upscaler from Microsoft :)

https://github.com/Blinue/Magpie/discussions/153

**Comment by hooke007** [comment]
@Blinue 我将关闭大量过期讨论，让一些典型的帖子更便于快速查找。
